wdl/finemap_sub.wdl

```diff
@@ -16,8 +16,12 @@ task ldstore {
     String n_samples_file = prefix + ".n_samples.txt"
     String zones
     String docker
+    Float snps = length(read_lines(zfile))
+    Array[Float] mem_coefficients
+    Int mem_ = floor(mem_coefficients[0]) + ceil(mem_coefficients[1]*snps)
```

Comment:
whats the point of this temp mem_

Reply:
Used to be there was a ceiling to at most X gigs of mem, but it seems I took that out since it kept failing some tasks.
Should be added back to limit it to 600 GB since we can't ask for machines with more memory (624 GB to be exact).